usr/share/rear/output/PXE/default/800_copy_to_tftp.sh

```diff
@@ -5,46 +5,43 @@
 # This file is part of Relax-and-Recover, licensed under the GNU General
 # Public License. Refer to the included COPYING for full text of license.
 
-if [[ ! -z "$PXE_TFTP_URL" ]] ; then
-    # E.g. PXE_TFTP_URL=nfs://server/export/nfs/tftpboot
-    local scheme=$( url_scheme $PXE_TFTP_URL )
+if [[ -n "$PXE_TFTP_UPLOAD_URL" ]] ; then
+    # E.g. PXE_TFTP_UPLOAD_URL=nfs://server/export/nfs/tftpboot
+    local scheme=$( url_scheme $PXE_TFTP_UPLOAD_URL )
 
     # We need filesystem access to the destination (schemes like ftp:// are not supported)
-    if ! scheme_supports_filesystem $scheme ; then
-        Error "Scheme $scheme for PXE output not supported, use a scheme that supports mounting (like nfs: )"
-    fi
+    scheme_supports_filesystem $scheme || Error "Scheme $scheme for PXE output not supported, use a scheme that supports mounting (like nfs: )"
 
-    mount_url $PXE_TFTP_URL $BUILD_DIR/tftpbootfs $BACKUP_OPTIONS
+    mount_url $PXE_TFTP_UPLOAD_URL $BUILD_DIR/tftpbootfs $BACKUP_OPTIONS
```

Comment:
```suggestion
    mount_url "$PXE_TFTP_UPLOAD_URL" $BUILD_DIR/tftpbootfs $BACKUP_OPTIONS
```

`BUIlD_DIR` is very very likely to be safe with regard to quoting